packages/hash/api/src/db/adapter.ts

```diff
@@ -545,11 +558,12 @@ export interface DbClient {
   /**
    * Create an aggregation for an entity.
    *
-   * @param params.sourceAccountId the account id of the source entity
-   * @param params.sourceEntityId the entity id of the source entity
-   * @param params.path the aggregation path
-   * @param params.operation the aggregation operation
-   * @param params.createdByAccountId the account id of the user that created the aggregation
+   * @param {string} params.sourceAccountId - the account id of the source entity
+   * @param {string} params.sourceEntityId - the entity id of the source entity
+   * @param {string} params.path - the aggregation path
+   * @param {object} params.operation - the aggregation operation
+   * @param {string} params.createdByAccountId - the account id of the user that created the aggregationfde
```

Comment:
I think you were going to flag here that it is equivalent to another field, as done for `deletedByAccountId`
```suggestion
   * @param {string} params.createdByAccountId - the account id of the user that created the aggregation (equivalent to the `appliedToSourceByAccountId`)
```

Reply:
Yes, thank you for catching